frontend/src/routes/Governance/policies/policy-details/PolicyDetailsPage.tsx

```diff
@@ -52,28 +53,18 @@ export function PolicyDetailsPage() {
         return policies[idx]
     }, [policies, policyName, policyNamespace])
 
-    const actions = useMemo(
-        () => [
-            {
-                id: 'edit-policy',
-                text: t('Edit policy'),
-                click: () =>
-                    history.push(
-                        NavigationPath.editPolicy.replace(':namespace', policyNamespace).replace(':name', policyName)
-                    ),
-                isAriaDisabled: true,
-                rbac: [
-                    selectedPolicy &&
-                        rbacPatch(
-                            selectedPolicy,
-                            selectedPolicy?.metadata.namespace ?? '',
-                            selectedPolicy?.metadata.name ?? ''
-                        ),
-                ],
-            },
-        ],
-        [selectedPolicy, policyNamespace, policyName, history, t]
-    )
+    const tableItem: PolicyTableItem = useMemo(() => {
+        const isExternal = resolveExternalStatus(selectedPolicy)
+        let source: string | JSX.Element = 'Local'
+        if (isExternal) {
+            const policySource = resolveSource(selectedPolicy, helmReleases, channels, subscriptions)
+            source = policySource ? getSource(policySource, isExternal, t) : 'Managed Externally'
```

Comment:
Will `'Managed Externally'` be translated?

Reply:
This is the same as https://github.com/stolostron/console/blob/23bfb388e7ddb3214ddb6cb5dc75833fdb815db8/frontend/src/routes/Governance/policies/Policies.tsx#L95, which is used for logic in the place like https://github.com/stolostron/console/blob/1f1e1b2e730e46ee37ca4146c5fa0711e942ada6/frontend/src/routes/Governance/components/PolicyActionDropdown.tsx#L107 , not for UI text.

Reply:
Why is `getSource` passed the `t` function? If `source` can contain a translated value, then this might break your logic.

Reply:
@KevinFCormier Good point. Translated function isn't need here for the internal logic, just reusing `getSource` function. So we may have two options:
1) update `getSource` function. let it be called without `t` function
2) update [PolicyActionDropdown.tsx](https://github.com/stolostron/console/blob/1f1e1b2e730e46ee37ca4146c5fa0711e942ada6/frontend/src/routes/Governance/components/PolicyActionDropdown.tsx#L107) as hasExternalResources: item.source !== t('Local')

Let me know your opinion

Reply:
Both updates will also fix the existing code [console/frontend/src/routes/Governance/policies/Policies.tsx](https://github.com/stolostron/console/blob/23bfb388e7ddb3214ddb6cb5dc75833fdb815db8/frontend/src/routes/Governance/policies/Policies.tsx#L95)

Reply:
> Why is `getSource` passed the `t` function? If `source` can contain a translated value, then this might break your logic.

I don't think in this case we will reach https://github.com/stolostron/console/blob/7fa0c313172ddf13a368111dd5bd0bf1ef76cbe4/frontend/src/routes/Governance/common/util.tsx#L357

Because at [console/frontend/src/routes/Governance/policies/Policies.tsx](https://github.com/stolostron/console/blob/23bfb388e7ddb3214ddb6cb5dc75833fdb815db8/frontend/src/routes/Governance/policies/Policies.tsx#L95) , if policySource is false, we get 'Managed Externally'. If policySource is true, we will get an element https://github.com/stolostron/console/blob/7fa0c313172ddf13a368111dd5bd0bf1ef76cbe4/frontend/src/routes/Governance/common/util.tsx#L373  which `t` function will not effect the logic at [console/frontend/src/routes/Governance/components/PolicyActionDropdown.tsx](https://github.com/stolostron/console/blob/1f1e1b2e730e46ee37ca4146c5fa0711e942ada6/frontend/src/routes/Governance/components/PolicyActionDropdown.tsx#L107)

Here I passed  `t` function just to satisfied the parameters requirement of `getSource` function